### DISCUSSION TITLE
Add Live Links to Project Examples on Website

### DISCUSSION THREAD

**Discussion by Suraj1089** [zauberzeug/nicegui#1466] [Ideas / Feature Requests]
Currently, the NiceGUI website showcases project examples, but it would be incredibly helpful to include live links (if available) for these projects. This would allow users to easily interact with the showcased projects without having to clone the repository and run them locally.

**Proposed Solution:**
- Add an iframe or link to each project example on the website if it is deployed and has a live version with preview.
- Provide a brief description of each project, along with its live link, making it easier for users to explore and interact with these projects.

This enhancement would improve the overall user experience and make it more convenient for users to explore the library's capabilities through real-world examples.

Example:[ Streamlit Gallery page](https://streamlit.io/gallery?category=data-visualization)
![Screenshot from 2023-08-22 00-12-15](https://github.com/zauberzeug/nicegui/assets/85509795/d87511a4-5eb7-4755-ba07-9731c1b08fc8)

**Comment by markospy** [comment]
This is okay. Having a visual support helps to compare the project with others similar by exposing in a direct way its potentialities.

**Comment by rodja** [comment]
Integrating screenshots would be very nice -- but a bit time consuming to maintain. Maybe someone wants to create a pull request?
Running live examples is much more tricky and costly. We would need to deploy them on multiple servers around the world to minimize latency.
